test/tests.js

```diff
@@ -224,13 +224,13 @@ exports = module.exports = {
                         'content-type': 'image/x-icon'
                     }
                 },
-
+                
```

Comment:
You actually added them which isn't allowed.

Reply:
ooops^^